### PR TITLE
JWST spectra ingest, spectra utils improvements

### PR DESCRIPTION
**Pull request by kelle**
Ingesting one JWST T dwarf spectrum.
Also working on improving spectra ingest scripts and modifying given new `astrodb_scripts` package. 

For data ingests:

- [x] includes script used for ingest.
- [x] includes modified JSON files
- [x] Add new tests
- [x] Update the Versions table

### CODE REVIEW

**Comment by kelle** [comment]
Apologies, this is really two different things.

- the reorg of the ingest scripts used for spectra.  these are mostly directory name and file name changes. Also removed functions which are now in the `astrodb_scripts` package

- an ingest of one new spectrum.  @SherelynA , check out the updated JSON file for W1935.

**Comment by kelle** [comment]
I guess I could add some tests for `ingest_spectrum` function...

File: scripts/ingests/ingest_jwst_spectra.py [hunk 1]
```diff
@@ -20,6 +20,13 @@
     mode="FS",
 )
 
+ingest_publication(
+    db,
+    publication="Fahe24",
+    description="W1935-1546 aurora Nature paper",
```
Comment (SherelynA) [inline scripts/ingests/ingest_jwst_spectra.py:26]: I think we can add the title for the paper @kelle

File: scripts/ingests/ingest_jwst_spectra.py [hunk 2]
```diff
@@ -26,7 +26,7 @@
     spectrum=file,
     regime="mir",
```
Comment (SherelynA) [inline scripts/ingests/ingest_jwst_spectra.py:34]: Regime is the wavelength coverage? Shouldn't it be nir instead of mir @kelle

**Comment by SherelynA** [comment]
@kelle do you also plan to ingest the MIRI JWST photometry?

**Comment by kelle** [comment]
I do not have those data. If you want to post here, I'll give it a go at ingesting.

File: scripts/ingests/ingest_jwst_spectra.py [hunk 3]
```diff
@@ -0,0 +1,45 @@
+from astrodb_scripts import load_astrodb, ingest_instrument, ingest_publication
```
Comment (dr-rodriguez) [inline scripts/ingests/ingest_jwst_spectra.py:1]: I wonder at the filename `ingest_jwst_spectra` and what it's doing. Is this a script we want to continuously update as we ingest more? 
If this is a one-off, should it be `ingest_jwst_Fahe24` or something like that? 
If it's meant for single spectra and users just change the individual settings, should it be `ingest_single_spectrum`?

Comment (kelle) [reply scripts/ingests/ingest_jwst_spectra.py:1]: I will rename the script to be a one off.

**Comment by SherelynA** [comment]
> I do not have those data. If you want to post here, I'll give it a go at ingesting.

This is the data: We used the following filters and the magnitudes/uncertainties (aper_total_vegamag) used :MIRI.F1000W - (13.740374118847015,  0.00509224851727859)  &   MIRI.F1280W - (13.125575556995827, 0.00711303904668107)  & MIRI.F1800W - (12.108791667404923, 0.01817644357837899)

**Comment by kelle** [comment]
- Photometry (and tests) added
- more ingest script work

**Comment by kelle** [comment]
This could be reviewed and merged at any point. I'm just gonna keep polishing the new scripts and tests until it's merged. :)

File: documentation/PhotometryFilters.md [hunk 4]
```diff
@@ -12,6 +12,7 @@ Columns marked with an asterisk (*) may not be empty.
 | width | width of band | Angstroms (10<sup>-10</sup>m) | |
 
 ## Notes
+- Useful functions: `ingest_photometry_filter`, `fetch_svo`, `assign_ucd`
```
Comment (dr-rodriguez) [inline documentation/PhotometryFilters.md:15]: Not clear where these functions are defined, are these all `utils.simple`? Sounds like not for all of them.

Comment (kelle) [reply documentation/PhotometryFilters.md:15]: These are gonna move to `astrodb_scripts` so I'd like to leave vague for now.

**Review by dr-rodriguez — APPROVED** [review]

**Comment by SherelynA** [comment]
> I assume those are JY, right? And any reason to maintain all those significant figures?

@kelle  I don't know if I understand this well. I gave the magnitudes, not the fluxes which like you mentioned are in Janskys.

**Comment by kelle** [comment]
Yes, I reminded myself about magnitudes being flux ratios after posting that comment.  I rounded the numbers based on the uncertainties. What numbers are reported in the paper?

**Comment by SherelynA** [comment]
> Yes, I reminded myself about magnitudes being flux ratios after posting that comment. I rounded the numbers based on the uncertainties. What numbers are reported in the paper?

@kelle The numbers reported in the paper are the following F1000W: (13.740±0.005) & F1280W: (13.126±0.007) & F1800W: (12.107±0.017).

File: data/cwisep_j193518.58-154620.3.json [hunk 5]
```diff
@@ -35,6 +35,48 @@
             "epoch": null,
             "comments": null,
             "reference": "Meis20.74"
+        },
```
Comment (SherelynA) [inline data/cwisep_j193518.58-154620.3.json:38]: F1000W: (13.740±0.005) & F1280W: (13.126±0.007) & F1800W: (12.107±0.017).

**Review by SherelynA — CHANGES_REQUESTED** [review]
The only thing that needs to be changed are the magnitudes. I put the ones we used in the paper.

**Review by SherelynA — APPROVED** [review]

**Pull request merged**